tests/ExamplePluginWithLib/CMakeLists.txt

```
@@ -45,10 +45,11 @@ macro(addLocalExternalFolder NAME FOLDER )
                 -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
                 -DCMAKE_PREFIX_PATH=${PLUGIN_DEMO_DIRECTORY}/lib/cmake
                 -DRadium_DIR=${Radium_DIR}
-                -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
-                -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-                -DCMAKE_CC_COMPILER=${CMAKE_CC_COMPILER}
-                -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DQt5_DIR=${Qt5_DIR}
+        -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+        -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+        -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Don't change the indentation..
```suggestion
                -DQt5_DIR=${Qt5_DIR}
                -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
                -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
                -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
                -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

---

formatted according to clang-format !

---

So apply it on the surrounding lines as well